### DISCUSSION TITLE
v2.0.8

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2717] [分享 / Share]
如果你是从 v1 过来的, 记得看下 v2 的发布公告: https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.7

- 修复 v1 中选择不再提示后变成弹另一种提示. (#2693)
- 修复 `自动隐藏侧栏` 在右侧的触发区域. (虽然还是不建议设置在右侧的时候用这个功能) (#2647)
- 合集包安装完后会刷新其他组件的安装状态了.
- 自定义顶栏:
  - 限制了 `动态` 等弹窗的高度, 以免无法触发下一页的加载. (#2472)
  - 修复 `历史` 面板看了没有封面的直播间后报错. (#2550)
  - `消息` 和 `稍后再看` 支持实时刷新. (#2689)
- 新增插件 `快捷键扩展 - 夜间模式`, 可以为夜间模式配置快捷键了. (#2556)

其他功能请求近期先不接了, 让我把清爽首页写完先 (新建文件夹

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.8'>v2.0.8</a>.</em>

**Comment by 921j** [comment]
感谢开发者大佬的辛苦更新！
不过对于我来说，其他的功能倒是可以等，但 默认播放器模式-宽屏 这个功能希望能早点修复
特别是中键打开新视频页等下再切过去的情况下不生效，非得刷新才行，没办法，装了一个另外的 默认宽屏 的脚本顶着用了……

**Reply by littleButterflies** [reply]
这个功能V1 旧版本 没有失效 哦

**Reply by ChaXxl** [reply]
你说的这个功能是啥啊，我都没用过。
我最喜欢的功能还是简化首页+夜间模式，其余的不重要

**Comment by StqGo** [comment]
感谢大佬更新！辛苦
但好像#2689 稍后再看还是不能实时更新诶

**Reply by the1812** [reply]
记得把自定义顶栏也更新下

**Reply by StqGo** [reply]
收到！已经可以了  万分感谢！

**Comment by Lee-7723** [comment]
清爽首页！坐等

**Reply by xzk-kokodayo** [reply]
期待

**Reply by Revicess** [reply]
等出来再更新了

**Reply by Malus-Mill** [reply]
确实、当初就是清爽主页入的

**Reply by woshijiangyi** [reply]
这个对我超级重要。借楼感谢开发大佬们

**Reply by sorker** [reply]
是的，现在没有了都不想看哔哩哔哩了

**Comment by FantasticAiming** [comment]
哇噢噢噢！感谢作者大大的夜间模式快捷键 幸苦了！爱了爱了！

**Reply by FantasticAiming** [reply]
键盘操作爱好者体验感拉满了~~  太感谢了！！

**Reply by ChaXxl** [reply]
为啥还要快捷键，难道不是一直开着夜间模式的吗？

**Reply by FantasticAiming** [reply]
> 

我还是更习惯日间 但在某些场景下就开深色  比如 视频总体为黑色等情况

**Comment by kkss2020** [comment]
希望大佬发布个自动发弹幕的功能

**Reply by Geeyun-JY3** [reply]
这很容易滥用成制造 spam 啊🤔

**Reply by 921j** [reply]
> 这很容易滥用成制造 spam 啊thinking

B站好像有10s的限制，我只需要打开视频就自动发送一条就行了：）
我现在用的是 autofill 的扩展，只能自动填一条弹幕但并不能自动发送
另外有个脚本能自动发射直播弹幕
[独轮车-说书人自动弹幕发射器](https://greasyfork.org/zh-CN/scripts/396285-%E7%8B%AC%E8%BD%AE%E8%BD%A6-%E8%AF%B4%E4%B9%A6%E4%BA%BA%E8%87%AA%E5%8A%A8%E5%BC%B9%E5%B9%95%E5%8F%91%E5%B0%84%E5%99%A8)
这个脚本好像又太复杂了，像真是为 spam 而备的，而且只能在直播页面用，不能在视频页面用。

**Reply by Geeyun-JY3** [reply]
为什么会需要打开视频发一条弹幕……

**Reply by 921j** [reply]
> 为什么会需要打开视频发一条弹幕……

对创作者来说，弹幕会给他们制作的视频增加一点热度吧，就像 github 的 star 一样；
对于我们这些观众来说，也需要留下自己类似 “到此一游” 的足迹一样，表示 “我看过了”、“我收到了”、“你的创作有人看”之类的心情……

**Reply by the1812** [reply]
这些行为都是需要经过考虑的，star 是你喜欢这个项目才点，弹幕是你想吐槽那一刻的内容而发；不应该由脚本去自动生成

**Reply by Geeyun-JY3** [reply]
这种「到此一游」的行为，在我看来和抢楼行为无异（类比到弹幕的话就是满屏的「热乎」），都是在制造无意义信息。

更何况 B 站连 YouTube 回溯自己发过的所有评论的功能都没有（早砍了），更困难的找回发过的所有弹幕更是无从说起，日后也就不可能通过留下过的「到此一游」溯找回看过的所有视频。（这倒是让我联想到有人想参考 Last.fm 的思路，「scrobble」所有看过的视频。）
而且近期 GitHub 的 star 就在改，变成可以把仓库加进某个可自定义的 list 中，反倒是变得有点像收藏夹了。

至于给创作者的反馈，我觉得这不是可持续且健康的反馈方式，过多的此类行为更像是在制造泡沫。不然也没必要设计出点赞投币收藏发弹幕发评论这么多种不同的反馈形式了，就是进行区分的。

**Reply by 921j** [reply]
> 这些行为都是需要经过考虑的，star 是你喜欢这个项目才点，弹幕是你想吐槽那一刻的内容而发；不应该由脚本去自动生成

感谢大佬，不过我倒觉得弹幕不仅仅有吐槽的功能，打招呼也应是其功能之一，虽然只是一句简简单单的“来了来了”，我如果是视频发布者，也会乐意看到自己的视频有这些……当然手打显得更诚实些，但有时经常忘记了，现在生活都是快节奏，哪怕只是形式做做样子，也比啥都不做强些吧。
另外抱歉的是我回到 Bilibili-Evolved v1 了，因为很多我觉得离不开的功能比如宽屏功能不正常、隐藏搜索推荐也不能去掉那一行字，与 bilibili live helper 也不兼容等……只能等 v2 版本正常了或 v1 彻底不能用了再升级吧，只是深色模式有问题的话可以配合 dark reader 来用了。

**Reply by 921j** [reply]
> 这种「到此一游」的行为，在我看来和抢楼行为无异（类比到弹幕的话就是满屏的「热乎」），都是在制造无意义信息，即 spam。
> 
> 更何况 B 站连 YouTube 回溯自己发过的所有评论的功能都没有（早砍了），更困难的找回发过的所有弹幕更是无从说起，日后也就不可能通过留下过的「到此一游」溯找回看过的所有视频。（这倒是让我联想到有人想参考 Last.fm 的思路，「scrobble」所有看过的视频。） 而且近期 GitHub 的 star 就在改，变成可以把仓库加进某个可自定义的 list 中，反倒是变得有点像收藏夹了。
> 
> 至于给创作者的反馈，我觉得这不是可持续且健康的反馈方式，过多的此类行为更像是在制造泡沫。不然也没必要设计出点赞投币收藏发弹幕发评论这么多种不同的反馈形式了，就是进行区分的。

你和大佬的话都是站在开发者的绝对理智角度来说的，我看到的是Up们都希望大家多多发弹幕（哪怕大部分只是无意义的）、留言，我个人觉得弹幕、留言、收藏、投币都各有其功能，弹幕本身就是个 spam 收容器，情绪共鸣地，留言则是真正需要理智参与的，收藏分享投币表示喜欢，除了空降功能，大部分人也并没指望通过曾发过的弹幕来回溯曾看过的视频，但也正因为弹幕功能，才让这种形式的视频形式比过了以往那种“安静”观影的模式，可能大部分人平常就是这样喜欢这样乱哄哄地一起看热闹，而只有少数时间比如看电影时才会想要安安静静一些吧。

**Reply by Geeyun-JY3** [reply]
隐藏原版顶栏搜索推荐这功能 v2 好像就没有……

**Reply by 921j** [reply]
> 隐藏原版顶栏搜索推荐这功能 v2 好像就没有……

装了之后在  “全部”-“搜索” 里搜不到，但在 “插件”-“已安装的插件” 里能看到 “搜索栏 - 搜索推荐” 后面只有个回收站的图标

**Reply by Geeyun-JY3** [reply]
建议你再认真看一次功能说明：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#搜索栏---搜索推荐

**Reply by 921j** [reply]
> 

哦，我装这个插件的时候没注意看说明，装了之后还以为是功能失效了，原来不装它才是像 v1 那样“隐藏”，多谢提醒！

**Comment by ChaXxl** [comment]
## 大家可以请作者大佬喝杯奶茶，这样简化首页就能很快出来了
https://github.com/the1812/Bilibili-Evolved/blob/preview/doc/donate.md

**Comment by Congee31** [comment]
Safari iPad无法正常使用，无法在视频界面拉动进度条，只是点击进度条进行跳转相应时间点！全屏等多项功能时常无效点击，控制条有时候点不出来，点击进入直播室，无法进入直播室，显示网址无效！

**Reply by the1812** [reply]
本来就不支持 iPad

**Reply by FoundTheWOUT** [reply]
what, ipad可以安装脚本吗，你是怎么安装的

**Comment by woshijiangyi** [comment]
谢谢大佬！！！
坐等清爽页面

**Comment by the1812** [comment]
发个进度(
最上面的 4 个板块基本完成了, 剩余视频分区 / 番剧时间表 / 排行榜

![image](https://user-images.githubusercontent.com/26504152/146676602-c38a40be-6c3e-4404-80e4-0cb9c5bb5ac3.jpg)

在 v2 中, 每个板块可以自由调整位置

![image](https://user-images.githubusercontent.com/26504152/146676697-9aff213b-38c0-42b1-a0e4-b1560cd00a71.jpg)

**Reply by ChaXxl** [reply]
# 作者大佬不用太赶的，平时注意休息，保重身体。
要是没什么事的话等会就发更新吧
![](http://i0.hdslb.com/bfs/emote/3087d273a78ccaff4bb1e9972e2ba2a7583c9f11.png)

**Reply by the1812** [reply]
半成品顶多发到预览版, 做完估计得明年了(

**Reply by littleButterflies** [reply]
**beta 版本 怕是要 20222 上线了。大佬保重身体，有钱了请你喝茶**

**Reply by ShuiHuo** [reply]
能增加设置直角和无边框样式吗

**Reply by cxw620** [reply]
这个是真的好看

**Comment by Congee31** [comment]
有计划支持吗？iPad使用上感觉不错，就是个别功能有点问题！

发自我的iPad

**Reply by the1812** [reply]
没有

**Comment by xzk-kokodayo** [comment]
官方又出了新版首页，貌似是win11的风格
![哔哩哔哩-゜-゜-つロ-干杯-bilibili](https://user-images.githubusercontent.com/69005158/147213813-28fda1eb-93ab-4131-bde2-41d4b252e1f2.png)

**Reply by xzk-kokodayo** [reply]
但感觉罗列按钮有点丑

**Reply by Geeyun-JY3** [reply]
新版首页这个样不挺久了么（#2209

**Reply by xzk-kokodayo** [reply]
> 新版首页这个样不挺久了么（#2209

这样吗，才给我推送新首页

**Comment by hd4357** [comment]
更新到了2.0.8了 为什么置顶栏没有变成以前的那样 的？

**Reply by Geeyun-JY3** [reply]
装了 `自定义顶栏` 组件没有
